frontend/templates/product/sections/ProductSection/index.tsx

```diff
@@ -112,26 +106,50 @@ export function ProductSection({
                <div>
                   <List spacing="2.5" fontSize="sm" mt="5">
                      <ListItem display="flex" alignItems="center">
-                        <ListIcon
-                           as={FaPercent}
-                           color="brand.500"
-                           boxSize="5"
-                        />
+                        <Flex
+                           justifyContent="center"
+                           alignItems="center"
+                           w={5}
+                           h={5}
+                           mr={1.5}
+                        >
+                           <FaIcon
+                              icon={faBadgeDollar}
+                              h="4"
+                              color="brand.500"
+                           />
+                        </Flex>
```

Comment:
This feels weird, because I'd have figured we'd want the `ListIcon` component still.

Reply:
We can achieve the same visual result with listIcon. I've reverted to it